gui/src/renderer/components/ParameterInput.js

```diff
@@ -223,15 +224,15 @@ const ParameterInput = ({
                 <Stack spacing={0.5}>
                     <Typography sx={{ fontWeight: 'bold' }}>{parameter.name}</Typography>
                     <Typography sx={{ fontFamily: 'monospace' }}>{`[${parameter_group_id}.${parameter.id}]`}</Typography>
```

Comment:
What do you think about moving this into the header? it's easy to miss at the end of the tooltip